.circleci/copy-configs

```diff
@@ -16,14 +16,16 @@ readonly config_bucket='amplify-circleci-android-payload'
 # So, use two simple arrays and join on index.
 # sources are remote URLs, targets are local file paths
 readonly sources=(
-    "s3://$config_bucket/awsconfiguration.json"
+    "s3://$config_bucket/awsconfiguration-analytics.json"
+    "s3://$config_bucket/awsconfiguration-api.json"
```

Comment:
This change is included in the IAM REST PR.

Once it goes in, we can just delete the `awsconfiguration.json`. At the moment is a copy of `awsconfiguration-analytics.json`.